bootloader/src/pyi_pythonlib.c

```diff
@@ -280,18 +293,6 @@ pyi_wargv_from_argv(int argc, char ** argv)
     return wargv;
 }
 
-void
-pyi_free_wargv(wchar_t ** wargv)
-{
-    wchar_t ** arg = wargv;
-
-    while (arg[0]) {
-        free(arg[0]);
-        arg++;
-    }
-    free(wargv);
-}
```

Comment:
Because `pyi_free_wargv()` is needed in the function above: `pyi_wargv_from_argv()` and there is no prototype for `pyi_free_wargv()`.

As there are also no prototypes for the other functions in this file, I thought it would look silly to add one prototype for this function only.

If you feel like you would prefer me to add a prototype for `pyi_free_wargv()` and revert its location just let me know.